.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
 <!-- Please use this line to close one or multiple issues when this pullrequest gets merged
 You can add another line right under the first one:
-resolves #1234
-resolves #1235
+resolves #1234 or closes #1234 or fixes #1234
+resolves #1235 or closes #1235 or fixes #1235
```

Comment:
One variant is enough.
Moreover, one line is enough.
Lets avoid asking contributors to remove a lot of unnecessary text.